packages/rollup-plugin/test/projects/stc-config/stylable.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports.stcConfig = {
```

Comment:
maybe use the function it will protect this code from changes...then add ts-check